### DISCUSSION TITLE
BitBucket

### DISCUSSION THREAD

**Discussion by dvelopin** [thebaselab/codeapp#300] [Q&A]
Hello,

Thank you for working on this app. I purchased it on the Apple App Store. I use VS Code all the time on my laptop, so this is a welcomed app to my inventory.

I know this app is still being developed, but is it possible to use this with BitBucket, or is this strictly GitHub? Are you looking to expand this to other platforms in the future?

Thank you so much!

**Comment by bummoblizard** [comment]
Hi. You can use the app with any git remote including Azure, Gitlab and others.

**Reply by dvelopin** [reply]
Oh very cool! I assumed it was just GitHub because you can only specify one set of remote credentials. Does this mean you need the same username and password across platforms? Or can you somehow add credentials for each respective platform?

**Reply by bummoblizard** [reply]
That would be a nice feature request! I will see how I can implement this.

**Reply by dvelopin** [reply]
That would be cool. Thank you! Another feature request would be to have multiple author identities. I have multiple jobs and have to use different accounts for each job. Although I know you can set this in the .git config file, so perhaps it already respects this?

Thanks again, looking forward to using this app more
